url-shortener-db/url_shortener_db/models.py

```diff
@@ -0,0 +1,52 @@
+from datetime import datetime
+
+from sqlalchemy import BigInteger, Column, Integer, String
+from sqlalchemy.ext.declarative import declarative_base
+from url_shortener_db.base_62_convertion import Base62Convertion
+from url_shortener_db.utils import create_session
+
+Base = declarative_base()
+session = create_session()
+
+
+class URL(Base):
+    __tablename__ = "URL"
+    id = Column(Integer, primary_key=True)
+    custom_id = Column(BigInteger, nullable=False)
```

Comment:
Why do we need this?

Reply:
I store custom_id in db to make search by short_url faster. I decode short_url to integer, and search not by short_url but by custom_id, because search by Integer is faster than by Varchar also I've created index for custom_id column.

Reply:
Looks like `custom_id` or `(custom_id, short_url)` pair can be PK. What do you think?